src/sentry/web/frontend/vercel_extension_configuration.py

```diff
@@ -1,32 +1,13 @@
 from __future__ import absolute_import
 
 
-from sentry.integrations.pipeline import IntegrationPipeline
-from sentry.web.frontend.base import BaseView
 from sentry import features
+from .integration_extension_configuration import IntegrationExtensionConfigurationView
 
 
-class VercelExtensionConfigurationView(BaseView):
-    auth_required = False
+class VercelExtensionConfigurationView(IntegrationExtensionConfigurationView):
+    provider = "vercel"
+    external_provider_key = "vercel"
```

Comment:
when would these be different?

Reply:
@MeredithAnya for VSTS, the value of `external_provider_key` is `vsts-extension`